parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -330,6 +330,14 @@ ensureKubelet() {
     systemctlEnableAndStart kubelet || exit $ERR_KUBELET_START_FAIL
 }
 
+ensureLabelNodes() {
```

Comment:
just to make sure I understand the intention here, this script runs for all kubernetes versions on master nodes _only_ to and adds labels to _both_ master and agent nodes?

Reply:
Yes, that's the intent.

Reply:
@CecileRobertMichon can you think of a clean way only to start the `label-nodes.sh` service on Kubernetes 1.16 and later? It's safe to run on any cluster, but it's basically a no-op on earlier clusters.

Reply:
I'm going to try to do this using the `${KUBERNETES_VERSION}` env var.